cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html

```diff
@@ -201,7 +201,7 @@
     }, {
         'titles': ['Deputy General Counsel', 'Assistant Director'],
         'slug':   'litigation-and-oversight',
-        'name':   'Vacant',
+        'name':   '(continuing) Vacant',
```

Comment:
"(continuing)" was just for us -- to let us know that the position is *still* vacant even with these updates. It should not be published.